Create ui-c8y-1021-49-2-users-in-latin-america-cant-access-cumulocity-locale-is-es-419.md (#3039)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-49-2-users-in-latin-america-cant-access-cumulocity-locale-is-es-419.md

---
date: ""

> title: Fixed issue preventing users in Latin America with locale es-419 from accessing Cumulocity

Reviewer: ```suggestion
title: Users with locale es-419 can again access the Cumulocity platform
```

> title: Users in Latin America cant access Cumulocity - Locale is es-419 - seems to cause problem. (#8408) [GRAFT][release/cd] (#8436)

Reviewer 2: ```suggestion
title: Fixed issue preventing users in Latin America with locale es-419 from accessing Cumulocity
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62820
version: 1021.49.2
---

> Users in Latin America cant access Cumulocity - Locale is es-419 - seems to cause problem. (#8408) [GRAFT][release/cd] (#8436)

Reviewer 2: ```suggestion
Some users in Latin America were unable to access Cumulocity when their locale was set to es-419. This issue has been resolved and users with the es-419 locale can now successfully access and use the Cumulocity platform again. The fix ensures that users in the affected region have uninterrupted access to Cumulocity and can utilize all its features as expected.
```

> Some users in Latin America were unable to access Cumulocity when their locale was set to es-419. This issue has been resolved and users with the es-419 locale can now successfully access and use the Cumulocity platform again. The fix ensures that users in the affected region have uninterrupted access to Cumulocity and can utilize all its features as expected.

Reviewer: ```suggestion
Some users in Latin America were unable to access {{< product-c8y-iot >}} when their locale was set to es-419. This issue has been resolved and users with the es-419 locale can now successfully access and use the {{< product-c8y-iot >}} platform again. The fix ensures that users in the affected region have uninterrupted access to {{< product-c8y-iot >}} and can utilize all its features as expected.
```
